lib/runner.js

```diff
@@ -50,21 +50,21 @@ function repositoryToNativePackageName(repository/*:string*/) {
   //
   // matches[1] = "rtfeldman"
   // matches[2] = "node-test-runner"
-  var matches = repository.match(/\/([^\/]+)\/([^\/\.]+)\.git/);
+  var matches = repository.match(/\/([^\/]+)\/([^\/]+)\.git/);
 
   if (matches instanceof Array) {
-    var username = matches[1];
+    var userName = matches[1];
     var repoName = matches[2];
 
-    if (typeof username === "string" && typeof repoName === "string") {
```

Comment:
I checked that as we require the matching groups to match at least one symbol each at this point both `userName` and `repoName` are not empty strings. Regular expression guarantees that this check is always true if there is a match:

```javascript
"https://github.com/rtfeldman/node-test-runner.git".match(regex)
(3) ["/rtfeldman/node-test-runner.git", "rtfeldman", "node-test-runner", index: 18, input: "https://github.com/rtfeldman/node-test-runner.git"]
"https://github.com//node-test-runner.git".match(regex)
null
"https://github.com/rtfeldman/.git".match(regex)
null
"https://github.com/123/node-test-runner.git".match(regex)
(3) ["/123/node-test-runner.git", "123", "node-test-runner", index: 18, input: "https://github.com/123/node-test-runner.git"]
```